tests/test_cli.sh

```diff
@@ -66,8 +69,9 @@ if [ -z "$CLI" ]; then
     ./src/sbin/lorax-composer --sharedir $SHARE_DIR $BLUEPRINTS_DIR &
 else
     export PACKAGE="composer-cli"
+    systemctl stop lorax-composer
     setup_tests /usr/share/lorax /var/lib/lorax/composer/blueprints
-    systemctl restart lorax-composer
+    systemctl start lorax-composer
```

Comment:
This is an unrelated change, should better be in a separate commit. How does this change behaviour?

Reply:
It is related to the previous commit. I've squashed the two and now it is more evident why we need this change.